assets/stylesheets/openqa.scss

```diff
@@ -64,6 +64,16 @@ $color-step-actions: #666;
     overflow: hidden;
 }
 
+.col-lg-4.text-nowrap {
+    display: flex;
+    align-items: baseline;
+}
+
+.col-lg-4.text-nowrap span {
+    font-size: 17px;
```

Comment:
if this is to align with the font size of h4, better use $font-size-h4 - that's the reason we use sass